cmake/Modules/LocalJpeg.cmake

```diff
@@ -35,7 +35,7 @@ else()
         GIT_REPOSITORY "https://github.com/libjpeg-turbo/libjpeg-turbo.git"
         GIT_TAG "${AVIF_JPEG_TAG}"
         LIST_SEPARATOR |
-        BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --config $<CONFIG> --target jpeg-static
+        BUILD_COMMAND ${CMAKE_COMMAND} --build $<BINARY_DIR> --config $<CONFIG> --target jpeg-static
```

Comment:
The original code is correct according to CMake documentation:
https://cmake.org/cmake/help/latest/module/ExternalProject.html

Search for "BINARY_DIR" in that page.

So something else is wrong.

Reply:
Reverting